wr_config.yml

```diff
@@ -379,6 +378,13 @@ clouddns: "8.8.4.4,8.8.8.8"
 # OpenStack.
 cloudos: "Ubuntu Xenial"
 
+# containerimage:  What container image should be used for spawned pods? This
+# defaults to "ubuntu:latest". It is overridden by the --container_image option
+# to `wr k8s deploy` and the --cloud_os option of `wr manager start`.
+#
+# This option is only relevant when you are using the kubernetes scheduler
```

Comment:
full stop. This is a good place to warn users about any gotchas there may be when selecting a container image to use. "has bash" is one? "allows sudo"? I don't know, whatever the requirements are. Where do these names come from? Can they be looked up or searched for somewhere? Mention that here as well.

Reply:
The names in short hand (as above) default to docker hub image:tag pairs, you can also reference a different public registry: e.g `gcr.io/kubeflow-images-public/tensorflow-1.8.0-notebook-cpu:v0.2.1` will pull from gcr, quay.io also works.

If I get time to do secrets management sensibly then private registries wil aosl work.